src/driver/test/service-architecture/SecondaryAllowed.yml

```diff
@@ -0,0 +1,57 @@
+SchemaVersion: 2018-07-01
+Pool:
+    QueryOptions:
+        maxPoolSize: 1000
+        socketTimeoutMS: 5000
+        connectTimeoutMS: 5000
+Actors:
+    - Name: StepdownCommand
+      Type: AdminCommand
+      Threads: 1
+      ExecutionStrategy:
+          Retries: 0
+          ThrowOnFailure: false
+      Phases:
+        - NoOp: true
+        - Phase: 1
+          Repeat: 1
+          Operation:
+              OperationMetricsName: "Stepdown"
+              OperationName: AdminCommand
+              OperationSleepBefore: 30 seconds
+              OperationCommand:
+                  replSetStepDown: 60
+              OperationAwaitStepdown: yes
+    - Name: SecondaryPreferredRead
+      Type: RunCommand
+      Threads: 100
+      ExecutionStrategy:
+          Retries: 0
+          ThrowOnFailure: false
+      Database: test
+      Phases:
+        - NoOp: true
+        - Phase: 1
+          Operation: &find_one
+              OperationMetricsName: "Find"
+              OperationName: RunCommand
+              OperationIsQuiet: yes
+              OperationMinPeriod: 100 microseconds
+              OperationCommand:
+                  find: Collection0
+                  readConcern: { level: local }
+                  $readPreference: { mode: secondary }
+    - Name: Loader
+      Type: Loader
+      Threads: 1
+      Database: test
+      CollectionCount: 1
+      DocumentCount: 100
+      BatchSize: 100
+      Document:
+          x: {$randomint: {min: 0, max: 20}}
```

Comment:
Just stumbled across this myself, I think this now needs to be `^RandomInt`. (See [here](https://github.com/mongodb/genny/commit/fb7bc6a2d2b9406328ea67c6a461b4d713e751cf))